frontend/public/components/app-contents.tsx

```diff
@@ -24,12 +24,6 @@ import {
   PageSectionVariants,
 } from '@patternfly/react-core';
 
-// React Router's proptypes are incorrect. See https://github.com/ReactTraining/react-router/pull/5393
-(Route as any).propTypes.path = PropTypes.oneOfType([
```

Comment:
This was causing problems with `yarn run build`. Issue appears to be fixed with newer version of library.